templates/controllers/tab/settings/contextIndexing/form/contextIndexingForm.tpl

```diff
@@ -31,6 +31,10 @@
 		{fbvFormSection label="manager.setup.customTags" description="manager.setup.customTagsDescription"}
 			{fbvElement type="textarea" multilingual="true" id="customHeaders" name="customHeaders" value=$customHeaders}
 		{/fbvFormSection}
+		{fbvFormSection label="manager.setup.searchEngineIndexing.sitemap"}
+			{url|assign:"sitemapPath" router=$smarty.const.ROUTE_PAGE page="sitemap"}
+			<label class="description">{translate key="manager.setup.searchEngineIndexing.sitemapDescription" path=$sitemapPath}</label>
```

Comment:
Use a regular `<p>` here or if that doesn't look try just a `<div>`. A `<label>` should only be used when associated with a form field, otherwise this can play havoc with screen readers.